lib/narn-lib.js

```diff
@@ -49,8 +70,16 @@ exports.getNpmArgs = narnArgs => {
       npmTarget = "init";
       npmArgs = [];
       break;
+    case "prune":
```

Comment:
`yarn prune` is a valid command (even though it isn't necessary to run it in yarn projects), so I'm okay with this one.